Source/WebKit/UIProcess/XR/PlatformXRSystem.cpp

```diff
@@ -146,6 +211,22 @@ void PlatformXRSystem::sessionDidUpdateVisibilityState(XRDeviceIdentifier device
     });
 }
 
+void PlatformXRSystem::setImmersiveSessionState(ImmersiveSessionState state)
+{
+    RELEASE_LOG_DEBUG(XR, "PlatformXRSystem: Immersive state update: Old=%d, New=%d", (int)m_immersiveSessionState, (int)state);
```

Comment:
Do we want to ship with this logging? (Dean always asked me this question)

Reply:
I'll remove it.